File: prime_counter.js
function isPrime(num){
    for(let i=2; i<=(num/2); i++){
        if(!(num%i)){
            return 0;
        }
    }
    return 1;
}

function main (){
    let numPrimes=0;

    for(let i = 2;i<250001; i++){
        numPrimes +=isPrime(i);
    }

    console.log(numPrimes);
}

main();

/**
 $ time node ./prime_counter.js 
22044

real    0m9.892s
user    0m0.046s
sys     0m0.077s
 */

File: prime_counter.py
def isPrime(n):
    for i in range(2,n//2+1):
        if(not (n%i)):
            return 0
    return 1

numPrimes=0
for i in range(2,250001):
    numPrimes+=isPrime(i)
print(str(numPrimes))

'''
$ time python ./prime_counter.py
22044

real    1m49.135s
user    0m0.047s
sys     0m0.015s
'''